router/gossip_test.go

```diff
@@ -22,11 +22,8 @@ type mockChannelConnection struct {
 // for Routes and GossipSenders.
 func NewTestRouter(name string) *Router {
```

Comment:
The comment above this needs updating since we now certainly do have a bunch of goroutines.